uploader.json

```diff
@@ -3335,6 +3335,11 @@
             "symbolic_name": "amq_broker",
             "pattern": []
         },
+        {
+            "glob": "/boot/config-*",
+            "symbolic_name": "kernel_config",
+            "pattern": []
+        },
```

Comment:
Glob files won't work for the V1 client in `uploader.json`, you need to add a `file` spec instead in the form `"/boot/()config-.*"`.

Reply:
Done